idioms-json-2.1-custom/campaign-v-actors.json

```diff
@@ -53,6 +58,36 @@
             "spec_version": "2.1",
             "target_ref": "identity--2d1c6ab3-5e4e-48ac-a32b-f0c01c2836a8",
             "type": "relationship"
+        },
+                {
+            "created": "2014-08-08T15:50:10.983Z",
+            "id": "relationship--f1b3f586-3a25-497a-8cdf-493d95ac3561",
+            "modified": "2014-08-08T15:50:10.983Z",
+            "relationship_type": "attributed-to",
+            "source_ref": "incident--229ab6ba-0eb2-415b-bdf2-079e6b42f51e",
```

Comment:
I think I see the same issue as the STIX 2.0 version of the bundle. Maybe these relationships should not be created?